src/libraries/TokenIdUtil.sol

```diff
@@ -13,6 +13,14 @@ import "../config/errors.sol";
  *  | tokenType (24 bits) | productId (40 bits) | expiry (64 bits) | longStrike (64 bits) | shortStrike (64 bits) |
  *  * ------------------- | ------------------- | ---------------- | -------------------- | --------------------- *
  */
+
+/**
+ * Short Token ID =
```

Comment:
maybe rename to shorten Token id, just so it won't be confusing with "short vs long" short.
( I think I referred to shortTokenId vs longTokenId several times in the codebase already)

Reply:
calling it Compressed